docs/querying/api.md

```diff
@@ -1031,6 +1031,34 @@ $ curl http://localhost:9090/api/v1/status/tsdb
 
 *New in v2.15*
 
+### WAL Replay Stats
```

Comment:
Yeah, I was wondering what release this would fall into. I think it's probably `v2.26`, although I'm not exactly sure how your release cycles work and there's already an rc out.

Reply:
Since it's a new feature and not a bug fix, it'd go only into 2.27, since the 2.26 RC is already cut.